src/Agent.Worker/TestResults/ResultsCommandExtension.cs

```diff
@@ -391,5 +400,6 @@ internal static class PublishTestResultsEventProperties
         public static readonly string RunTitle = "runTitle";
         public static readonly string PublishRunAttachments = "publishRunAttachments";
         public static readonly string ResultFiles = "resultFiles";
+        public static readonly string TestRunSystem = "testRunSystem";
```

Comment:
TestRunSystem

Reply:
I thought it would be better and consistent if I follow like other properties.

Reply:
this is different from other as in this case this is predefined custom field.

Reply:
Yeah, makes sense. But here it's an even property which will be coming from node. So still having it as is. But when creating the field, having the right case, as you suggested.